src/database/migrations/V1__addUsers.ts

```diff
@@ -0,0 +1,14 @@
+import Knex from 'knex';
+
+export async function up(db: Knex) {
+  await db.schema.createTable('example_users', (table) => {
+    table.bigIncrements('id');
+    table.timestamps(true, true);
+    table.string('email').unique().notNullable();
+    table.string('hashed_password').notNullable();
```

Comment:
I guess this is an example right? We are not having passwords

Reply:
Yes, it's an example for creating migrations.